apps/election-manager/src/components/ExportElectionBallotPackageModalButton.tsx

```diff
@@ -0,0 +1,321 @@
+              <h1>Export Ballot Package</h1>
+              <p>
+                <USBImage src="usb-drive.svg" alt="Insert USB Image" />
+                Would you like to export the ballot configuration now? A zip
+                archive will automatically be saved to the default location on
+                the mounted USB drive. Optionally, you may pick a custom export
+                location.
+              </p>
+            </Prose>
+          )
+          break
+        }
+      }
+      break
+
+    case 'RenderBallot': {
+      actions = (
+        <LinkButton onPress={closeModal} disabled>
+          Cancel
+        </LinkButton>
+      )
+      const {
+        ballotStyleId,
+        precinctId,
+        contestIds,
+        isLiveMode,
+        locales,
+      } = state.currentBallotConfig
+      const precinctName = getPrecinctById({ election, precinctId })!.name
+
+      mainContent = (
+        <Prose>
+          <h1>
+            Generating Ballot{' '}
+            {state.ballotConfigsCount - state.remainingBallotConfigs.length} of{' '}
+            {state.ballotConfigsCount}…
+          </h1>
+          <ul>
+            <li>
+              Ballot Style: <strong>{ballotStyleId}</strong>
+            </li>
+            <li>
+              Precinct: <strong>{precinctName}</strong>
+            </li>
+            <li>
+              Contest count: <strong>{contestIds.length}</strong>
+            </li>
+            <li>
+              Language format:{' '}
+              <strong>{getHumanBallotLanguageFormat(locales)}</strong>
+            </li>
+            <li>
+              Filename:{' '}
+              <Monospace>{state.currentBallotConfig.filename}</Monospace>
+            </li>
+          </ul>
+          <HandMarkedPaperBallot
+            ballotStyleId={ballotStyleId}
+            election={election}
+            electionHash={electionHash}
+            isLiveMode={isLiveMode}
+            precinctId={precinctId}
+            onRendered={onRendered}
+            locales={locales}
+          />
+        </Prose>
+      )
+      break
+    }
+
+    case 'ArchiveEnd': {
+      actions = (
+        <LinkButton onPress={closeModal} disabled>
+          Cancel
+        </LinkButton>
+      )
+      mainContent = (
+        <Prose>
+          <h1>Finishing Download…</h1>
+          <p>
+            Rendered {pluralize('ballot', state.ballotConfigsCount, true)},
+            closing zip file.
+          </p>
+        </Prose>
+      )
+      break
+    }
+
+    case 'Done': {
+      if (usbDriveStatus !== UsbDriveStatus.recentlyEjected) {
+        actions = (
+          <React.Fragment>
+            <LinkButton onPress={closeModal}>Cancel</LinkButton>
+            <USBControllerButton primary small={false} />
+          </React.Fragment>
+        )
+      } else {
+        actions = <LinkButton onPress={closeModal}>Close</LinkButton>
+      }
+      mainContent = (
+        <Prose>
+          <h1>Download Complete!</h1>
```

Comment:
No need for exclaiming 😄  let's save this for when we really need it. 
```suggestion
          <h1>Download Complete</h1>
```